test-extender.sh

```diff
@@ -54,6 +77,37 @@ docker run \
   -v $PWD/cnb/ext/:/cnb/ext \
   -v $PWD/cnb/buildpacks/:/cnb/buildpacks \
   -u root \
-  test-builder \
-  /cnb/lifecycle/extender kaniko build ubuntu:bionic
+  $REGISTRY_HOST/test-builder \
+  /cnb/lifecycle/extender kaniko build "$REGISTRY_HOST/test-builder"
+  #              args:    <kaniko|buildah> <build|run> <base-image>
+
+echo ">>>>>>>>>> Running extend on run image..."
+ # TODO: we should probably not have to mount /cnb/ext? Can we copy the static Dockerfiles to layers?
```

Comment:
That sounds like a good idea to me. The lifecycle currently just leaves the Dockerfiles where the extensions wrote them, but copying them sounds like a logical next step.